### PR TITLE
initial pop-up commit

### PR DESCRIPTION
**Pull request by Writerman**
_Complete the form below and remove this line._

### Pop-up task

[Demo](https://writerman.github.io/pop-up) |
[Code base](https://github.com/Writerman/pop-up)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by madmaxWMFU** [comment]
Please correct your demo link)

**Comment by Writerman** [comment]
Did I do it right?

File: submissions/Writerman/my_pop_up/index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+        <li class="menu_item">
+          <a href="#" class="menu_link">
+              <span class="menu_icon icon_User"></span>
+              <span class="app_name">Stats</span>
+          </a>
+        </li>
+
+      </ul>
+    <main class="main">
```
Comment (madmaxWMFU) [inline submissions/Writerman/my_pop_up/index.html:100]: Tag `main` should situated in `body`. 
The tag main is unclosed.

File: submissions/Writerman/my_pop_up/index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                <span class="app_icon icon_third"></span>
+                <span class="app_name">third</span>
+            </a>
+
+            <input
+                  type="checkbox"
+                  class="button_extra_input"
+                  id="button_extra_input"
+                >
+                <label
```
Comment (madmaxWMFU) [inline submissions/Writerman/my_pop_up/index.html:59]: Add a focus for this element

File: submissions/Writerman/my_pop_up/index.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            name="toggle_pop_up"
+            id="toggle_pop_up"
+            class="toggle_input"
+          >
+          <label
+          for="toggle_pop_up"
+          class="toggle_label">
+          <span class="menu_icon icon_book"></span>
+        </label>
+        <div class="pop_up">
```
Comment (madmaxWMFU) [inline submissions/Writerman/my_pop_up/index.html:36]: Make sure the links doesn't overlap with menu.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good job, see some comments below

**Comment by Writerman** [comment]
I fix it.

**Comment by madmaxWMFU** [comment]
Tag `main` must be under tag `nav`.

**Review by madmaxWMFU — APPROVED** [review]

File: submissions/Writerman/my_pop_up/css/style.css [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+  font-size: 16px;
+  color: #444;
+}
+.nav {
+  padding: 20px;
+  position: absolute;
+  top: 3.0em;
+  right: 2em;
+  width: 350px;
+  border: 2px solid #666666;
```
Comment (miroslav-kolomiets) [inline submissions/Writerman/my_pop_up/css/style.css:23]:
```suggestion
  border: 2px solid #666;
```
If you use in other places short syntax, use it everywhere ;)

File: submissions/Writerman/my_pop_up/css/style.css [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+}
+
+.icon_User {
+  background-image: url(../images/User.png);
+  width: 50px;
+  height: 50px;
+}
+
+.icon_Defend
+{
```
Comment (miroslav-kolomiets) [inline submissions/Writerman/my_pop_up/css/style.css:76]:
```suggestion
.icon_defend {
```

File: submissions/Writerman/my_pop_up/css/style.css [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+}
+.menu_link:hover >  .icon_Atack{
+  box-shadow: 10px -10px 22px 10px #000
+}
+.menu_link:focus > .icon_Atack  {
+  opacity:0.5;
+  box-shadow: 10px -10px 22px 10px #000
+}
+
+.icon_User {
```
Comment (miroslav-kolomiets) [inline submissions/Writerman/my_pop_up/css/style.css:69]:
```suggestion
.icon_user {
```

File: submissions/Writerman/my_pop_up/css/style.css [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+  height: 50px;
+}
+
+.icon_Defend
+{
+  background-image: url(../images/Defend.png);
+  width: 50px;
+  height: 50px;
+}
+.icon_Atack {
```
Comment (miroslav-kolomiets) [inline submissions/Writerman/my_pop_up/css/style.css:81]:
```suggestion
.icon_atack {
```

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Nice job! But something can be improved.
Снимок экрана 2020-11-14 в 12 11 45 AM

**Comment by miroslav-kolomiets** [comment]
Hi! @Writerman, do you need any help with this?

**Comment by Writerman** [comment]
I've fixed it.

File: submissions/Writerman/my_pop_up/css/style.css [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+  background-repeat: no-repeat;
+  background-position-x: center;
+  background-position-y: bottom;
+  justify-self: center;
+}
+.app_name {
+  display: block;
+  text-align: center;
+}
+.toggle_input {
```
Comment (A-Ostrovnyy) [inline submissions/Writerman/my_pop_up/css/style.css:162]: Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkboxes in the right way

File: submissions/Writerman/my_pop_up/css/style.css [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+  height: 50px;
+}
+.pop_up {
+  display: none;
+    position: absolute;
+    top: 7em;
+    right: 0.6em;
+    width: 350px;
+    border: 2px solid #666;
+    background: #fff;
```
Comment (A-Ostrovnyy) [inline submissions/Writerman/my_pop_up/css/style.css:91]: Use [Prettier](https://prettier.io/) for formatting your code.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Nice work!
See my comment above!

File: submissions/Writerman/my_pop_up/css/style.css [hunk 10]
```diff
@@ -0,0 +1,239 @@
...
+  cursor: pointer;
+  background-color: greY;
+}
+.button_extra_input:hover + .button_extra_label,
+.button_extra_input:focus + .button_extra_label{
+  color: #03c;
+  text-decoration: underline;
+  opacity: 0.5;
+}
+.icon_first {
```
Comment (miroslav-kolomiets) [inline submissions/Writerman/my_pop_up/css/style.css:222]: This is also bad practice. Here and below. Better to name the icons essentially. Describing what the icon actually is.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
NIce job. But something still needs to be done.

**Comment by OleksiyRudenko** [comment]
Closing this PR for a long time inactivity.
@Writerman  please feel free reopening this PR once you have updates based on reviewers' comments.

**Pull request closed without merging**